src/schema/types/group.graphql

```diff
@@ -0,0 +1,19 @@
+type Group {
+  id: ID!
+  product: Product!
+  type: TypeEnum!
```

Comment:
When renaming the enum, do not rename this field. As a property, it's ok to just call it "type".

Reply:
Done